### PR TITLE
Regression - gmail.com hangs on login (fix #526)

### LINKED ISSUE #526: Regression - gmail.com hangs on login

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @inikulin @churkin @VasilyStrelyaev

### CODE REVIEW

File: test/client/fixtures/utils/url-test.js [hunk 1]
```diff
@@ -412,3 +412,20 @@ test('"base" tag with an href attribute that is set to a relative url (GH-422)',
 
     base.parentNode.removeChild(base);
 });
+
+asyncTest('resolving url after write "base" tag (GH-526)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/utils/url-test.js]:
```
asyncTest('resolving url after writing the "base" tag (GH-526)', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
ping @inikulin

**Comment by inikulin** [comment]
lgtm

**Pull request merged**